spoti-vote/src/components/Header.jsx

```diff
@@ -1,7 +1,8 @@
 import React, {Component} from 'react';
-import logo from '../img/sbahn.png'
+import logo from '../img/spotiLogo.png';
```

Comment:
preferably a .SVG File

Reply:
The svg file has problems, it uses an S from a diffrent font than the one i designed it with, ill try to solve this problem tho